### PR TITLE
Bump OpenBLAS 0.3.9

### PR DESCRIPTION
**Pull request by haampie**
Following @ViralBShah's comment in the linear algebra channel on Slack, it might be the right time to bump OpenBLAS to 3.9.0 for the Julia 1.5 release.

- Remove Keno's Windows patch since it has been incorporated into OpenBLAS already
- Forward the `USE_BLAS_FFLAGS` to `OPENBLAS_FFLAGS`. This addresses an issue where OpenBLAS does not build LAPACK with `-fdefault-integer-8` and causes a segfault when running the OpenBLAS tests during the build (ref https://github.com/xianyi/OpenBLAS/issues/2429)
- ~Add the `-frecursive -fno-optimize-sibling-calls` flags to `USE_BLAS_FFLAGS`. I'm not 100% sure why this would be necessary, but read https://github.com/xianyi/OpenBLAS/issues/2429 where they include those flags.~ Actually, I'm inclined to drop this... from the comments there they mention that `-frecursive` cannot hurt but does not seem to be mandatory, and `-fno-optimize-sibling-calls` sounds very unrelated. The important bit is passing `-fdefault-integer-8` to LAPACK I figure.

Todo:

- [x] Make binary builder pull OpenBLAS 0.3.9 (https://github.com/JuliaPackaging/Yggdrasil/pull/621)

New todo:

- [x] Wait for a new Yggdrasil release of OpenBLAS (https://github.com/JuliaPackaging/Yggdrasil/pull/987)

### CODE REVIEW

File: deps/blas.mk [hunk 1]
```diff
@@ -46,9 +46,11 @@ $(BUILDDIR)/$(OPENBLAS_SRC_DIR)/build-compiled: | $(BUILDDIR)/objconv/build-comp
 endif
 endif
 
-OPENBLAS_FFLAGS := $(JFFLAGS)
+OPENBLAS_FFLAGS := $(JFFLAGS) $(USE_BLAS_FFLAGS)
 OPENBLAS_CFLAGS := -O2
 
+# OPENBLAS_FFLAGS += -frecursive -fno-optimize-sibling-calls -fdefault-integer-8
```
Comment (staticfloat) [inline deps/blas.mk]: This looks like something that slipped through. ;)

**Review by staticfloat — APPROVED** [review]
Overall looks good, but I know Keno was working on the win32 patch very recently, so let's see what he says.  Once he weighs in, we can merge the Yggdrasil PR, get binaries uploaded, update the BB release and checksums, then merge this.

File: deps/blas.mk (hunk 1 above)
Comment (haampie) [reply deps/blas.mk]: Yup, thanks for pointing out!

**Comment by haampie** [comment]
> but I know Keno was working on the win32 patch very recently, so let's see what he says

On that note, from https://github.com/JuliaLang/julia/pull/34923#issue-381630976

>  If this works, I'll need to include the openblas patch in the julia

I think the Yggdrasil and Julia base patches are still not in sync at the moment.

**Comment by haampie** [comment]
Seems like the tests still pass on Ubuntu 18.04 with this patched version of OpenBLAS:
```
Step #1: Step 6/6 : RUN cd /julia && make test-LinearAlgebra
...
Step #1: Test Summary: |  Pass  Total
Step #1:   Overall     | 91305  91305
Step #1:     SUCCESS
```
so that looks promising.

**Comment by Keno** [comment]
> * Remove Keno's Windows patch since it has been incorporated into OpenBLAS already

Not the case. Part of it is definitely still needed.

**Comment by haampie** [comment]
Ok, so now the Windows patch is back and is identical to the one at https://github.com/JuliaPackaging/Yggdrasil/pull/621

@staticfloat last difference between here and Yggdrasil is this change https://github.com/JuliaPackaging/Yggdrasil/commit/b572419a1d907778025a4afb32cda50c9fb656af should the "Patch out `-Ofast` on power" patch also be included here?

**Comment by staticfloat** [comment]
Yes, we still need to carry that patch both here and in Yggdrasil.

**Comment by haampie** [comment]
Ok, so now everything is in sync with the Yggdrasil PR

**Review by staticfloat — APPROVED** [review]

**Comment by haampie** [comment]
Ok, so now that the BB OpenBLAS version is bumped we can finally see if it works with Julia :crossed_fingers: 

I've also bumped LAPACK to 3.9.0, but I don't really understand why it's still in the Makefile since it is almost always shipped with OpenBLAS?

**Comment by ViralBShah** [comment]
Thank you @haampie

**Comment by ViralBShah** [comment]
LAPACK is separately in the Makefile for the use case where you don't use it bundled with OpenBLAS. Increasingly a rare use case but useful for architectures where OpenBLAS may not be available. 

It should usually be the version of LAPACK that ships with the OpenBLAS we are using.

**Comment by haampie** [comment]
tester_linuxaarch64 has linalg errors:

```
LinearAlgebra/special: Test Failed at /buildworker/worker/tester_linuxaarch64/build/share/julia/stdlib/v1.5/LinearAlgebra/test/special.jl:126
  Expression: op(A, B) ��� op(Matrix(A), Matrix(B)) ��� Matrix(op(A, B))
   Evaluated: [0.9601490412773276 0.07897628661563409 ��� 6.626530213604507 4.990367748161503; 0.0 0.021276677108488847 ��� 4.391867807764307 5.003436245324435; ��� ; 0.0 0.0 ��� 0.38028646737079386 0.14890935339477882; 0.0 0.0 ��� 0.0 0.208343087340652] ��� [0.9601490412773276 0.07897628661563409 ��� 6.626530213604507 4.990367748161503; 0.0 0.021276677108488847 ��� 4.391867807764307 5.003436245324435; ��� ; 0.0 0.0 ��� 0.38028646737079386 0.14890935339477882; 0.0 0.0 ��� 0.0 0.208343087340652] ��� [0.9601490412773276 0.07897628661563409 ��� 6.626530213604507 4.990367748161503; 0.0 0.021276677108488847 ��� 4.391867807764307 5.003436245324435; ��� ; 0.0 0.0 ��� 0.38028646737079386 0.14890935339477882; 0.0 0.0 ��� 0.0 0.208343087340652]
Stacktrace:
 [1] record(::Test.DefaultTestSet, ::Union{Test.Error, Test.Fail}) at /buildworker/worker/package_linuxaarch64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:775
 [2] (::var"#44#53")() at /buildworker/worker/tester_linuxaarch64/build/share/julia/test/runtests.jl:337
 [3] cd(::var"#44#53", ::String) at ./file.jl:104
 [4] top-level scope at /buildworker/worker/tester_linuxaarch64/build/share/julia/test/runtests.jl:75
 [5] include(::Function, ::Module, ::String) at ./Base.jl:380
 [6] include(::Module, ::String) at ./Base.jl:368
 [7] exec_options(::Base.JLOptions) at ./client.jl:288
 [8] _start() at ./client.jl:490
LinearAlgebra/special: Test Failed at /buildworker/worker/tester_linuxaarch64/build/share/julia/stdlib/v1.5/LinearAlgebra/test/special.jl:126
  Expression: op(A, B) ��� op(Matrix(A), Matrix(B)) ��� Matrix(op(A, B))
   Evaluated: [0.5183657300754845 0.0 ��� 0.0 0.0; 0.0 0.6413577456911539 ��� 0.0 0.0; ��� ; 0.0 0.0 ��� 0.42311204165695826 0.0; 0.0 0.0 ��� 0.0 0.560892881044907] ��� [0.5183657300754845 0.0 ��� 0.0 0.0; 0.0 0.6413577456911539 ��� 0.0 0.0; ��� ; 0.0 0.0 ��� 0.42311204165695826 0.0; 0.0 0.0 ��� 0.0 0.560892881044907] ��� [0.5183657300754845 0.0 ��� 0.0 0.0; 0.0 0.6413577456911539 ��� 0.0 0.0; ��� ; 0.0 0.0 ��� 0.42311204165695826 0.0; 0.0 0.0 ��� 0.0 0.560892881044907]
Stacktrace:
 [1] record(::Test.DefaultTestSet, ::Union{Test.Error, Test.Fail}) at /buildworker/worker/package_linuxaarch64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:775
 [2] (::var"#44#53")() at /buildworker/worker/tester_linuxaarch64/build/share/julia/test/runtests.jl:337
 [3] cd(::var"#44#53", ::String) at ./file.jl:104
 [4] top-level scope at /buildworker/worker/tester_linuxaarch64/build/share/julia/test/runtests.jl:75
 [5] include(::Function, ::Module, ::String) at ./Base.jl:380
 [6] include(::Module, ::String) at ./Base.jl:368
 [7] exec_options(::Base.JLOptions) at ./client.jl:288
 [8] _start() at ./client.jl:490
Worker 5 failed running test LinearAlgebra/matmul:
Some tests did not pass: 966 passed, 2 failed, 0 errored, 0 broken.
LinearAlgebra/matmul: Test Failed at /buildworker/worker/tester_linuxaarch64/build/share/julia/stdlib/v1.5/LinearAlgebra/test/matmul.jl:603
  Expression: D ��� C
   Evaluated: [-26.861334345553136 -23.65816474235872 ��� -23.010518420790522 -24.981235209458834; -26.081022396434065 -25.08768970170442 ��� -22.715362284007657 -28.380158209617118; ��� ; -24.431823779625308 -22.50450114181978 ��� -22.61410724022273 -23.746031013418346; -24.151580246169104 -23.944203004528433 ��� -24.214649928719943 -24.591809064286803] ��� [-26.861334345553143 -23.65816474235872 ��� -23.01051842079052 -24.981235209458845; -26.081022396434054 -25.087689701704416 ��� -22.715362284007657 -28.380158209617115; ��� ; -24.431823779625308 -22.504501141819784 ��� -22.614107240222722 -23.746031013418346; -24.151580246169104 -23.944203004528433 ��� -24.214649928719954 -24.5918090642868]
Stacktrace:
 [1] record(::Test.DefaultTestSet, ::Union{Test.Error, Test.Fail}) at /buildworker/worker/package_linuxaarch64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:775
 [2] (::var"#44#53")() at /buildworker/worker/tester_linuxaarch64/build/share/julia/test/runtests.jl:337
 [3] cd(::var"#44#53", ::String) at ./file.jl:104
 [4] top-level scope at /buildworker/worker/tester_linuxaarch64/build/share/julia/test/runtests.jl:75
 [5] include(::Function, ::Module, ::String) at ./Base.jl:380
 [6] include(::Module, ::String) at ./Base.jl:368
 [7] exec_options(::Base.JLOptions) at ./client.jl:288
 [8] _start() at ./client.jl:490
LinearAlgebra/matmul: Test Failed at /buildworker/worker/tester_linuxaarch64/build/share/julia/stdlib/v1.5/LinearAlgebra/test/matmul.jl:609
  Expression: D ��� C
   Evaluated: [-26.861334345553136 -23.65816474235872 ��� -23.010518420790522 -24.981235209458834; -26.081022396434065 -25.08768970170442 ��� -22.715362284007657 -28.380158209617118; ��� ; -24.431823779625308 -22.50450114181978 ��� -22.61410724022273 -23.746031013418346; -24.151580246169104 -23.944203004528433 ��� -24.214649928719943 -24.591809064286803] ��� [-26.861334345553143 -23.65816474235872 ��� -23.01051842079052 -24.981235209458845; -26.081022396434054 -25.087689701704416 ��� -22.715362284007657 -28.380158209617115; ��� ; -24.431823779625308 -22.504501141819784 ��� -22.614107240222722 -23.746031013418346; -24.151580246169104 -23.944203004528433 ��� -24.214649928719954 -24.5918090642868]
Stacktrace:
 [1] record(::Test.DefaultTestSet, ::Union{Test.Error, Test.Fail}) at /buildworker/worker/package_linuxaarch64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:775
 [2] (::var"#44#53")() at /buildworker/worker/tester_linuxaarch64/build/share/julia/test/runtests.jl:337
 [3] cd(::var"#44#53", ::String) at ./file.jl:104
 [4] top-level scope at /buildworker/worker/tester_linuxaarch64/build/share/julia/test/runtests.jl:75
 [5] include(::Function, ::Module, ::String) at ./Base.jl:380
 [6] include(::Module, ::String) at ./Base.jl:368
 [7] exec_options(::Base.JLOptions) at ./client.jl:288
 [8] _start() at ./client.jl:490
Worker 11 failed running test LinearAlgebra/pinv:
Some tests did not pass: 286 passed, 2 failed, 0 errored, 0 broken.
LinearAlgebra/pinv: Test Failed at /buildworker/worker/tester_linuxaarch64/build/share/julia/stdlib/v1.5/LinearAlgebra/test/pinv.jl:92
  Expression: ���(norm(a * apinv * a - a) / norm(a), 0, atol = tol1)
   Evaluated: NaN32 ��� 0 (atol=1.0)
Stacktrace:
 [1] record(::Test.DefaultTestSet, ::Union{Test.Error, Test.Fail}) at /buildworker/worker/package_linuxaarch64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:775
 [2] (::var"#44#53")() at /buildworker/worker/tester_linuxaarch64/build/share/julia/test/runtests.jl:337
 [3] cd(::var"#44#53", ::String) at ./file.jl:104
 [4] top-level scope at /buildworker/worker/tester_linuxaarch64/build/share/julia/test/runtests.jl:75
 [5] include(::Function, ::Module, ::String) at ./Base.jl:380
 [6] include(::Module, ::String) at ./Base.jl:368
 [7] exec_options(::Base.JLOptions) at ./client.jl:288
 [8] _start() at ./client.jl:490
LinearAlgebra/pinv: Test Failed at /buildworker/worker/tester_linuxaarch64/build/share/julia/stdlib/v1.5/LinearAlgebra/test/pinv.jl:92
  Expression: ���(norm(a * apinv * a - a) / norm(a), 0, atol = tol1)
   Evaluated: NaN32 ��� 0 (atol=10000.0)
Stacktrace:
 [1] record(::Test.DefaultTestSet, ::Union{Test.Error, Test.Fail}) at /buildworker/worker/package_linuxaarch64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:775
 [2] (::var"#44#53")() at /buildworker/worker/tester_linuxaarch64/build/share/julia/test/runtests.jl:337
 [3] cd(::var"#44#53", ::String) at ./file.jl:104
 [4] top-level scope at /buildworker/worker/tester_linuxaarch64/build/share/julia/test/runtests.jl:75
 [5] include(::Function, ::Module, ::String) at ./Base.jl:380
 [6] include(::Module, ::String) at ./Base.jl:368
 [7] exec_options(::Base.JLOptions) at ./client.jl:288
 [8] _start() at ./client.jl:490
```

**Comment by ViralBShah** [comment]
If its only on aarch64, that suggests possible openblas bugs.

**Comment by haampie** [comment]
Is there any way to debug this easily? aarch64 docker image?

**Comment by ViralBShah** [comment]
I think @staticfloat may have something for you.

**Comment by staticfloat** [comment]
You.... can.... do an aarch64 docker image, but running everything inside of QEMU can be error-prone for things like this.

I suggest remoting into the aarch64 buildbot.  PM me on Slack with your SSH public key and I'll give you temporary access to an environment where I have a `julia` session open with the failing test already loaded up.  It's failing even in a build from source on the machine, so it's not a BB issue, it's most likely an issue with OpenBLAS itself.

**Comment by ViralBShah** [comment]
Any chance we can figure out what the issue is and get this in before code freeze?

**Comment by ViralBShah** [comment]
Of course, I have a bad feeling it is an openblas bug, and we'll need a new release.

**Comment by chriselrod** [comment]
OpenBLAS 0.3.9 is much faster than earlier versions on systems with AVX512 in my benchmarks, so I hope these issues can be solved.

Anyone have aarch64 to test?

**Comment by ViralBShah** [comment]
> Anyone have aarch64 to test?

Best option is to send your ssh key to @staticfloat and he can give you access to the buildbot.

**Comment by ViralBShah** [comment]
Using the julia nightly on aarch64 from yesterday, and doing a few different runs, most of the errors seem to be related to tolerance. The only one that seems worrisome is: 
```
LinearAlgebra/pinv: Test Failed at /home/sabae/julia-5cb3f3fbc1/share/julia/stdlib/v1.5/LinearAlgebra/test/pinv.jl:92
  Expression: ≈(norm(a * apinv * a - a) / norm(a), 0, atol = tol1)
   Evaluated: NaN32 ≈ 0 (atol=1.0)
```

**Comment by ViralBShah** [comment]
@haampie Can you rebase this to master?

**Comment by ViralBShah** [comment]
It only fails the first time! 
```
julia> test_pinv(a, m, n, tol1, tol2, tol3)
Test Failed at REPL[10]:4
  Expression: ≈(norm(a * apinv * a - a) / norm(a), 0, atol = tol1)
   Evaluated: NaN32 ≈ 0 (atol=1.0)
ERROR: There was an error during testing

julia> test_pinv(a, m, n, tol1, tol2, tol3)
Test Passed
````

**Comment by ViralBShah** [comment]
@chriselrod Would you be able to take a look? Maybe I am doing something wrong. Can you email/slack me your public key?

**Comment by haampie** [comment]
rebased & bumped binary builder openblas version to v0.3.9-2

**Comment by ViralBShah** [comment]
I would personally be ok to ship this as is. Is there a way to make aarch64 ignore the result of this test so it doesn't fail the CI? It is not a tier-1 architecture. 

We can open a separate issue and file it upstream etc when we have a good reproducer.

**Comment by ViralBShah** [comment]
We're now seeing more errors. I wonder if we can just bump to 0.3.9 on intel architectures.

**Comment by chriscoey** [comment]
> I would personally be ok to ship this as is. Is there a way to make aarch64 ignore the result of this test so it doesn't fail the CI? It is not a tier-1 architecture.
> 
> We can open a separate issue and file it upstream etc when we have a good reproducer.

Please. This would be really great to have in 1.5. I think it's overdue!

**Comment by ViralBShah** [comment]
> > I would personally be ok to ship this as is. Is there a way to make aarch64 ignore the result of this test so it doesn't fail the CI? It is not a tier-1 architecture.
> > We can open a separate issue and file it upstream etc when we have a good reproducer.
> 
> Please. This would be really great to have in 1.5. I think it's overdue!

@chriscoey You gave a thumbs down on the suggestion I made while simultaneously suggesting the opposite in this comment. Could you explain? 

Anyways, right now, unless someone can track down the issue, we're not going to be able to upgrade.

**Comment by chriscoey** [comment]
@ViralBShah I thumbs-upped the first suggestion (quoted) and thumbs-downed the second suggestion. I think it makes sense to merge given that aarch64 is not tier 1. But I am not on an Intel architecture, I'm on Ryzen.

**Comment by ViralBShah** [comment]
> @ViralBShah I thumbs-upped the first suggestion (quoted) and thumbs-downed the second suggestion. I think it makes sense to merge given that aarch64 is not tier 1. But I am not on an Intel architecture, I'm on Ryzen.

I see. I should clarify. I actually meant bump everything to openblas64 except for aarch64. But there are good reasons not to do this sort of thing piecemeal.

**Comment by staticfloat** [comment]
I'm pretty strongly against bifurcating OpenBLAS version by platform; it will undoubtedly result in someone writing something that only works on 1.5, then being surprised when that package works on Julia 1.5 x86, but fails when it runs on Julia 1.5 aarch64, and then is unable to express in their package compatibility that their package works on Julia 1.5 on non-aarch64.

Has anyone manage to bisect OpenBLAS to find the commit that breaks aarch64?

**Comment by ViralBShah** [comment]
I don't think anyone has attempted a bisect yet. I certainly haven't put in more time than finding the reproducer (which itself is a bit tricky).

**Comment by staticfloat** [comment]
I've bisected it to this commit: https://github.com/xianyi/OpenBLAS/commit/b7cc69ee622fed9039ab755b87eee9279d27d541

The next thing to try is to revert that in OpenBLAS on the v0.3.9 tag, and see if everything passes tests.  If so, we bake that patch into OpenBLAS, release it here, then tell upstream to fix that patch, hopefully with a reduced test case for them.  :)

**Comment by ViralBShah** [comment]
Let's file it right away. Producing a reduced test case had been a bit difficult.

**Comment by giordano** [comment]
Does https://github.com/xianyi/OpenBLAS/commit/c6af9bbb32469dcf9670343671469443629c7d21 help?

**Comment by ViralBShah** [comment]
Definitely worth trying it first before reverting the commit @staticfloat bisected to.

**Comment by giordano** [comment]
The commit is from https://github.com/xianyi/OpenBLAS/pull/2534, which fixes https://github.com/xianyi/OpenBLAS/issues/2496, and this issue was reportedly introduced by https://github.com/xianyi/OpenBLAS/pull/2340, which is the same PR Elliot bisected to.

**Comment by ViralBShah** [comment]
@haampie Would it be possible for you to apply the above patch to the openblas recipe and then update this PR?

**Comment by haampie** [comment]
it better work :D

**Comment by haampie** [comment]
Well, CI does not seem to like this PR. But the good news is that `LinearAlgebra/pinv` passes on aarch64.

Failures on mac:
> cat: /Users/julia/buildbot/worker/package_macos64/build/contrib/mac/startup.jl: No such file or directory

Failures on linux64, win32, win64 and aarch64 are all issues with Distributed

**Comment by ViralBShah** [comment]
It's not Distributed. The stacktrace always has that because all the parallel test processes are killed when there's a test failure.

Aarch64 is still showing several matmul test failures.
https://build.julialang.org/#/builders/28/builds/10506/steps/5/logs/stdio

All testers have this failure. Perhaps it is fixed on master and we can rebase.
```
      From worker 6:	Internal error: encountered unexpected error during compilation of undeclared_ceil:
      From worker 6:	ErrorException("Failed to parse LLVM Assembly: 
      From worker 6:	julia: llvmcall:3:18: error: use of undefined value '@llvm.ceil.f64'
      From worker 6:	%2 = call double @llvm.ceil.f64(double %0)
```

Mac failure seems to be a problem with hdiutil in assembling the final dmg because it can't access libpcre.

**Comment by haampie** [comment]
Oh, sorry, my bad. There were two commits for sgemm and dgemm resp that look almost identical, the patch only contains one of the two at the moment. Let me add it

**Comment by ViralBShah** [comment]
Bumped to BB_REL 4. Let's see how this goes.

**Comment by giordano** [comment]
I think you also need to regenerate the checksum files?

**Comment by ViralBShah** [comment]
> I think you also need to regenerate the checksum files?

Sigh. Yes.

**Comment by haampie** [comment]
> > I think you also need to regenerate the checksum files?
> 
> Sigh. Yes.

```
$ cat create_checksums.sh
#!/bin/bash

JULIA_ROOT="$1"
VERSION="$2"
BB="$3"
URL="https://github.com/JuliaBinaryWrappers/OpenBLAS_jll.jl/releases/tag/OpenBLAS-v$VERSION%2B$BB" 

# Download
curl -s "$URL" | grep -Eo "/JuliaBinaryWrappers/.*?/releases/.*?tar\.gz" | xargs -I{} wget https://github.com{}

# Create checksums
for f in *.tar.gz; do
    MD5=$(md5sum $f | awk '{ print $1 }')
    SHA512=$(sha512sum $f | awk '{ print $1 }')
    JULIA_NAME=$(basename "$f" | sed "s|v$VERSION|v$VERSION-$BB|g")
    DIR="$JULIA_ROOT/deps/checksums/$JULIA_NAME"

    mkdir -p "$DIR"
    echo "$MD5" > "$DIR/md5"
    echo "$SHA512" > "$DIR/sha512"
done

$ create_checksums.sh ~/project/julia 0.3.9 4
```

**Comment by haampie** [comment]
I think this is good news for linuxaarch64:

```
Test Summary:                       |     Pass  Error  Broken     Total
  Overall                           | 38658181      2  327766  38985949
...
    file                            |     1031      2              1033
...
```

So there's a test failing, but it is not openblas-related.

**Comment by giordano** [comment]
@haampie why not using [`contrib/refresh_bb_tarballs.sh`](https://github.com/JuliaLang/julia/blob/master/contrib/refresh_bb_tarballs.sh)? :sweat_smile:

**Comment by haampie** [comment]
I will from now on :laughing:

**Comment by ViralBShah** [comment]
Let's try rebasing on master.

**Comment by haampie** [comment]
Tests are green 🎉

**Comment by chriscoey** [comment]
Hooray! Thank you everyone! This currently has the backport 1.5 label, does that mean it won't be in 1.5.0? Forgive my ignorance

**Comment by StefanKarpinski** [comment]
That means it _will_ be in the 1.5 release (we also haven't branched yet).

**Comment by ViralBShah** [comment]
That was a lot of work for a version bump. Thanks @haampie! Taking off the label, since we have not yet branched. 

Is an openblas version update something we can expect in a 1.4 backport?

**Pull request merged**